build.sbt

```diff
@@ -2,14 +2,14 @@ lazy val root = (project in file("."))
   .settings(
     organization := "com.github.seratch",
     name := "bigquery4s",
-    version := "0.6",
-    scalaVersion := "2.12.3",
-    crossScalaVersions := Seq("2.12.3", "2.11.11", "2.10.6"),
+    version := "0.6-teads1",
```

Comment:
You don't need to modify the version. I will change this part after merging this PR.

Reply:
Sorry about that one, I made a custom build for my company's nexus to ensure it was working properly (and it does).